cli/build.rs

```diff
@@ -37,6 +38,15 @@ fn main() {
     } else {
         println!("cargo:rustc-env=JJ_VERSION={version}");
     }
+
+    let manifest_dir = PathBuf::from(std::env::var("CARGO_MANIFEST_DIR").unwrap());
+    let docs_symlink_path = manifest_dir.join("docs");
```

Comment:
nit: maybe we can assume that cwd is `CARGO_MANIFEST_DIR`? `GIT_HEAD_PATH` already relies on that.

Reply:
Good point that we already rely on that. Done.